lib/easy-stackup.js

```diff
@@ -0,0 +1,76 @@
+// simpler API main function
+'use strict'
+var gerberToSvg = require('gerber-to-svg')
+var shortId = require('shortid')
+var whatsThatGerber = require('whats-that-gerber')
+
+var pcbStackup = require('./index')
+
+module.exports = function(layers, optionsOrCallback, done) {
+  var options = {}
+
+  if (typeof optionsOrCallback === 'object') {
+    options = optionsOrCallback
+  }
+  else if (typeof optionsOrCallback === 'function') {
+    done = optionsOrCallback
+  }
+
+  if (!('id' in options)) {
```

Comment:
Can we do

``` js
options.id = options.id || shortId.generate()
```

It also catches the undesirable case of an empty string being passed, because `''` evaluates false (undesirable because the id is used as an internal prefix to ensure the IDs of individual pad shapes are unique and not accidentally used across layers